diff --git a/node_modules/@newswire/doc-to-archieml/index.js b/node_modules/@newswire/doc-to-archieml/index.js
index ca5c4d2..a2e085b 100644
--- a/node_modules/@newswire/doc-to-archieml/index.js
+++ b/node_modules/@newswire/doc-to-archieml/index.js
@@ -9,10 +9,19 @@ function readParagraphElement(element) {
   // sometimes it's not there, skip this all if so
   if (textRun) {
     // sometimes the content isn't there, and if so, make it an empty string
-    const content = textRun.content || '';
+    let content = textRun.content || '';
 
     // step through optional text styles to check for an associated URL
     if (!textRun.textStyle) return content;
+
+    // add inline style tags
+    if (textRun.textStyle.italic) {
+      content = content.replace(/([^\n]+)(\n)?/,'<i>$1</i>$2');
+    }
+    if (textRun.textStyle.bold) {
+      content = content.replace(/([^\n]+)(\n)?/,'<b>$1</b>$2');
+    }
+
     if (!textRun.textStyle.link) return content;
     if (!textRun.textStyle.link.url) return content;
 
