diff --git a/node_modules/parcel-bundler/src/Pipeline.js b/node_modules/parcel-bundler/src/Pipeline.js
index 6377259..e2d2829 100644
--- a/node_modules/parcel-bundler/src/Pipeline.js
+++ b/node_modules/parcel-bundler/src/Pipeline.js
@@ -27,7 +27,10 @@ class Pipeline {
       }
     } catch (err) {
       error = errorUtils.errorToJson(err);
-      error.fileName = path;
+      if (error === undefined)
+        console.log(err);
+      else
+        error.fileName = path;
     }
 
     return {
